spec.md

```diff
@@ -1074,18 +1074,18 @@ message NodePublishVolumeRequest {
   // REQUIRED.
   bool readonly = 6;
 
-  // End user credentials used to authenticate/authorize node
+  // Credentials used by Node plugin to authenticate/authorize node
   // publish request.
   // This field contains credential data, for example username and
   // password. Each key must consist of alphanumeric characters, '-',
   // '_' or '.'. Each value MUST contain a valid string. An SP MAY
   // choose to accept binary (non-string) data by using a binary-to-text
   // encoding scheme, like base64. An SP SHALL advertise the
   // requirements for credentials in documentation. COs SHALL permit
-  // users to pass through the required credentials. This information is
+  // passing through the required credentials. This information is
   // sensitive and MUST be treated as such (not logged, etc.) by the CO.
   // This field is OPTIONAL.
-  map<string, string> user_credentials = 7;
+  map<string, string> node_credentials = 7;
```

Comment:
As discussed in #140, `NodePublishVolume` might be called one per workload by a CO. Calling it `node_credentials` makes it sound like the credentials are for the entire node.

Reply:
@jdef before you had user_credentials which made sound like the same credential is used for everything.
controller_credential and node_credential are per function, the content of it depends on the usage of corresponding function controller or node. Making them even more granular is an option but imho is an overkill.